### ISSUE TITLE
Embed Code pop over is behind the footer

### ISSUE THREAD

**Issue by sosiology** [visualize-admin/visualization-tool#536]
**Describe the bug**
The embed code pop over is covered by the footer, which makes it hard to copy the embed code needed for AEM

**To Reproduce**
1. Go to https://int.visualize.admin.ch/en/v/nYQIQwAoYJeG
2. Click on Embed

**Expected behavior**
the pop over should be in the foreground not behind/ hidden by the footer

**Screenshots**
<img width="1339" alt="image" src="https://user-images.githubusercontent.com/29289679/165796187-7a779747-26a8-4491-a93e-73be24b83da4.png">

**Environment (please complete the following information):**
 - Visualize environment and version: INT v3.6.1 [a03b62e] 
 - Browser and version: FF, Version 99.0.1 (64-bit)

**Additional context**

**Comment by ptbrowne** [comment]
Fixed on int, I used popover from MUI now that have not the same problem so the look might be a little difference.

In addition to not having this z-index bug, MUI popovers try to place themselves in the current viewport, this is why if you do not scroll down to the bottom, the top of the popovers is not exactly at the same vertical place. If you scroll down to the bottom or dezoom the page, both the share and embed popovers will have enough space to display on the bottom and their top position will be the same.
